### DISCUSSION TITLE
Extracting Multiple Data Entries Using BAML

### DISCUSSION THREAD

**Discussion by octadion** [BoundaryML/baml#982] [Q&A] [answered]
I am currently working with BAML to extract structured data from a text response, but I'm facing a challenge when trying to extract multiple entries. For example, I have the following input:
```
#"
      John Doe

      Education
      - University of California, Berkeley
        - B.S. in Computer Science
        - 2020

      Skills
      - Python
      - Java
      - C++

John Doe2

      Education
      - University of California, Berkeley
        - B.S. in Computer Science
        - 2020

      Skills
      - Python
      - Java
      - C++    
"# 
```
In this case, I want to extract information for both "John Doe" and "John Doe2." However, my current implementation only extracts data for the first entry.

I have already searched the documentation but haven't found a solution to this issue. Could someone please advise on how to modify the extraction process to handle multiple entries? What approach or code adjustments would you recommend to ensure that all data entries are captured and returned as structured JSON outputs?

Thank you for your help!

**Comment by aaronvg** [comment, marked as answer]
Mind sharing your baml function signature?

You should only need to make the output an array: Resume[]

**Reply by octadion** [reply]
oh you're right, it worked, I didn't think to set it there, thanks
